### ISSUE TITLE
Ride Editor Copy Truncates LAT/LON precision

### ISSUE THREAD

**Issue by whitakma** [GoldenCheetah/GoldenCheetah#1074]
Having imported a fit file of a ride, I can view latitude and longitude data with full (too much?) precision in the edit ride tab, as expected.

I want to export this ride data so I can use the lat/long details to do some additional processing/analysis.

OPTION 1) Use the Activity/Export... menu to CSV. That does not work: Latitude, longitude (and temperature) are not included in the export - I'd expect all available columns to be included in the export.

OPTION 2) Cut and paste direct from the Ride Edit tab. That does not work either: Lat is 13dp Long 15dp, but once pasted these are truncated to 4 and 6 dp respectively.

By my maths a metre is equivalent to 0.000009 degrees of arc (longitude, at the equator etc), so I'd expect at least 6dp for the latitude, to match the longitude... At present, the rounded/truncated latitude obviously introduces significant errors into the position calcs (0.0001 degree = 11m, roughly).

OPTION 3) Use the Activity/Export... menu to TCK. This gives 9 and 10 dps for lat and long but the file format is inappropriate and needs post/processing faff to bring into a tabular form.

In short, can options 1 & 2 be fixed ?
